torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -184,12 +184,12 @@ Tensor norm_backward(Tensor grad, const Tensor & self, const optional<Scalar> &
   return norm_backward(grad, self, p_, norm);
 }
 
-Tensor pow_backward(Tensor grad, const Tensor & self, const Scalar & exponent_) {
-  auto exponent = (exponent_.isComplex()) ? exponent_.toComplexDouble() : exponent_.toDouble();
-  if (exponent == 0.0) {
+Tensor pow_backward(Tensor grad, const Tensor & self, const Scalar & exponent) {
+  if (exponent.equal(0.0)) {
     return at::zeros_like(self, LEGACY_CONTIGUOUS_MEMORY_FORMAT);
   } else {
-    auto out = grad * (exponent * self.pow(exponent - 1)).conj();
+    auto grad_lambda = [&](auto exp) { return grad * (exp * self.pow(exp - 1)).conj(); };
```

Comment:
Time to read up on auto type deduction...

Reply:
According to https://en.cppreference.com/w/cpp/language/lambda, `grad_lambda` is a generic lambda, which behaves like a template with one parameter. So this should work.

It's too bad that we can't actually write a test for this